Makefile

```diff
@@ -12,12 +13,17 @@ lint: sort style
 
 .PHONY: sort
 sort:
-	@echo "Checking imports with isort" && isort --check-only --diff .
+	@echo "Checking imports with isort" && isort --check-only --diff hypha
 
 .PHONY: sort-fix
 sort-fix:
-	@echo "Fixing imports with isort" && isort .
+	@echo "Fixing imports with isort" && isort hypha
 
 .PHONY: style
 style:
 	@echo "Checking code style with flake8" && flake8 .
+
+test: lint
```

Comment:
Can we move "lint" to its own line here to make it easier to read.

Reply:
I've updated the "test" target to use other smaller targets in https://github.com/HyphaApp/hypha/pull/2954/commits/a65345a42749c3ced5f9598b6632129bfdb0de47. 

The `test` target now looks like this
https://github.com/HyphaApp/hypha/blob/a65345a42749c3ced5f9598b6632129bfdb0de47/Makefile#L43-L44

This will be maintainable and easier to read.